src/puppetlabs/trapperkeeper/services/webrouting/webrouting_service.clj

```diff
@@ -38,6 +39,9 @@
   (get-route [this svc route-id]
              (core/get-route (service-context this) svc route-id))
 
+  (get-server [this svc route-id]
+             (core/get-server (service-context this) svc route-id))
+
```

Comment:
In the protocol you define two signatures, but here you only implement one of them, right?